src/contracts/treasury/Collector.sol

```diff
@@ -47,16 +48,26 @@ contract Collector is VersionedInitializable, ICollector, ReentrancyGuard {
    */
   mapping(uint256 => Stream) private _streams;
 
+  /**
+   * @notice Address of the current ACL Manager.
+   */
+  address internal _aclManager;
+
   /// @inheritdoc ICollector
   address public constant ETH_MOCK_ADDRESS = 0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE;
 
+  /**
+   * @notice FUNDS_ADMIN role granted in ACL Manager
+   */
+  bytes32 public constant FUNDS_ADMIN_ROLE = keccak256('FUNDS_ADMIN');
```

Comment:
why the `keccak256` - should fit without hashing, making it a bit more easy to fetch i'd say.